docs/internal/investigations/runtime-director-dapr/README.md

```diff
@@ -0,0 +1,101 @@
+# Runtime Director - Proof of concept
+
+This directory contains source code and Helm chart for Kyma Control Plane Runtime Director.
+
+Runtime Director 
+
+## Overview
+
+Runtime Director exposes simple REST API which returns Runtime configurations.
+The application periodically reloads configuration from a ConfigMap `runtime-director-config` in `kcp-poc` namespace. The configuration file has the following form:
+
+```yaml
+runtimes:
+  - id: "1"
+    data: # arbitrary data
+      foo: bar
+  - id: "2"
+    data: # arbitrary data
+      bar: baz
+```
+
+The application reloads configuration every 10 seconds. 
+
+## API
+
+The Application exposes the following endpoints:
+
+- `GET /runtimes`
+   
+    Returns configuration for all Runtimes.  
```

Comment:
Consider using indented bullet points here. But in this form, it looks fine as well.